### PR TITLE
[148] Replace YAML with typed objects

### LINKED ISSUE #148: Replace YAML files with typed objects for Storybook args
Consensus at 8/29 check-in is that we will use typed objects in place of YAML files for Storybook args. Args will live in a separate .ts or .tsx file that is imported into the story as the YAML is now.

For now, assume new args file will use the same args as the original one (aside from anything that needs to be fixed because it's missing or typed incorrectly). What should be an arg and whether or not it should match the controls for the end user is a question for another day.

### PR DESCRIPTION
**Pull request by kmonahan**
Closes #148 . Replaces all YAML files used for Storybook mock data with their typed object equivalents.

### CODE REVIEW

File: starterkits/drupal/examples/_components/views/FilmsListing.tsx
Comment (mrclay) [inline starterkits/drupal/examples/_components/views/FilmsListing.tsx:1]: Looks like a tooling error.

**Review by mrclay — APPROVED** [review]
Probably want to revert change in `starterkits/drupal/examples/_components/views/FilmsListing.tsx` but otherwise LGTM.

File: starterkits/drupal/examples/_components/views/FilmsListing.tsx
Comment (kmonahan) [reply starterkits/drupal/examples/_components/views/FilmsListing.tsx:1]: Oh yes, great catch!

**Comment by kmonahan** [comment]
Realized I need to update the component generator too

**Pull request merged**